OSDOCS-12183: adding how to delete manifests (#83246)
openshift/openshift-docs: modules/microshift-manifests-upgrade-scenario-ostree.adoc

// Module included in the following assemblies:
//
// * microshift/running_applications/microshift-deleting-resource-manifests.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-manifests-upgrade-scenario-ostree_{context}"]

> == Manifest upgrade scenario for OSTree systems

Reviewer: ```suggestion
= Upgrading manifests for OSTree systems
```

Author: done

> Use the following procedure in the upgrade scenario for OSTree systems where some resources need to be removed while retaining others to preserve data.

Reviewer: Blank line needed after title. Title needs to be at `=` level.

Author: done

> Use the following procedure in the upgrade scenario for OSTree systems where some resources need to be removed while retaining others to preserve data.

Reviewer: ```suggestion
Use the following procedure to remove some resources while retaining others to preserve data.
```

Author: done

[IMPORTANT]
====
For OSTree systems, the `delete` subdirectories are read only.
====

.Procedure

. Identify the manifest that needs updating.

> . Create new manifest for resource deletion that needs to be placed in the `delete` subdirectories.

Reviewer 2: Should a new manifest be created ? I think moving the existing manifest to the delete directory ensures that the proper resource is removed ?

Author: as per our discussion I am resolving this comment. @dhensel-rh

> . Create new manifest for resource deletion that needs to be placed in the `delete` subdirectories.

Reviewer: ```suggestion
. Create a new manifest for resource deletion to place in the `delete` subdirectories.
```

Author: done

> . Create new manifest that needs to be applied in the manifest path. link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.

Reviewer: ```suggestion
. Create a new manifest to apply in the manifest path. link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.
```
See previous comment about using `access.redhat.com` links. Also, this sentence containing the cross reference seems to be missing a word at the beginning.

Author: done as per comment https://github.com/openshift/openshift-docs/pull/83246#discussion_r1817012102.

// Module included in the following assemblies:
//
// * microshift/running_applications/microshift-deleting-resource-manifests.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-manifests-upgrade-scenario-ostree_{context}"]
= Upgrading manifests for OSTree systems

Use the following procedure to remove some resources while retaining others to preserve data.

[IMPORTANT]
====
For OSTree systems, the `delete` subdirectories are read only.
====

.Procedure

. Identify the manifest that needs updating.
. Create a new manifest for resource deletion to be placed in the `delete` subdirectories.
. Create a new manifest to apply in the manifest path. See link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.

> . Use the procedure in "Removing manifests for OSTree systems" to create `delete` subdirectories and place the manifests created for resource deletion in this path.

Reviewer 3: The _Removing manifests for OSTree systems_ procedure doesn't seem to show how to "create `delete` subdirectories...". This would confuse me as a user, where I wouldn't be sure I was in the right place.

Author: I have changed the sentence.
